website/docs/reference/resource-configs/strategy.md

```diff
@@ -140,6 +226,24 @@ It's possible to implement your own snapshot strategy by adding a macro with the
 1. Create a macro named `snapshot_timestamp_with_deletes_strategy`. Use the existing code as a guide and adjust as needed.
 2. Use this strategy via the `strategy` configuration:
 
+<VersionBlock firstVersion="1.9">
+<File name='snapshots/<filename>.yml'>
+
+```yaml
+snapshots:
+  - name: my_custom_snapshot
+    relation: source('my_source', 'my_table')
+    config:
+      strategy: timestamp_with_deletes
+      updated_at: updated_at_column
+      unique_key: id
+      schema: snapshots
```

Comment:
I didn't try this out yet, but it looks right visually 👍

Reply:
I tested this out, and it worked:

```yaml
snapshots:
  - name: snapshot_name
    relation: ref("my_model")
    config:
      unique_key: id
      strategy: timestamp_with_deletes
      check_cols: all
```

Note that configuring `schema:` was not necessary, so we can simplify by removing that line.